examples/list_box_model/model/imp.rs

```diff
@@ -0,0 +1,36 @@
+//! Defines the implementation of our model
+
+use gio::subclass::prelude::*;
+use gtk::{gio, glib, prelude::*};
+
+use std::cell::RefCell;
+
+use crate::row_data::RowData;
+
+#[derive(Debug, Default)]
+pub struct Model(pub RefCell<Vec<RowData>>);
```

Comment:
```suggestion
pub struct Model(pub(super) RefCell<Vec<RowData>>);
```
People will copy/paste this, so better be safe than sorry